packages/engine-test-utils/src/__tests__/pods-core/GraphvizPod.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { tmpDir } from "@dendronhq/common-server";
+import { FileTestUtils, NoteTestUtilsV4 } from "@dendronhq/common-test-utils";
+import { GraphvizExportConfig, GraphvizExportPod } from "@dendronhq/pods-core";
+import fs from "fs-extra";
+import _ from "lodash";
+import path from "path";
+import { runEngineTestV5, WorkspaceOpts } from "../../engine";
+import { checkNotInString, checkString } from "../../utils";
+
+const setupBasic = async (opts: WorkspaceOpts) => {
+  const { wsRoot, vaults } = opts;
+  await NoteTestUtilsV4.createNote({
+    wsRoot,
+    vault: vaults[0],
+    fname: "parent",
+    body: [`[[foo.linked]]`].join("\n"),
+  });
+  await NoteTestUtilsV4.createNote({
+    wsRoot,
+    vault: vaults[0],
+    fname: "parent.linked",
+    body: [`[[foo]]`].join("\n"),
+  });
+  await NoteTestUtilsV4.createNote({
+    wsRoot,
+    vault: vaults[0],
+    fname: "unlinked",
+    body: [`[[parent]]`].join("\n"),
+  });
+};
+
+describe("graphviz export pod", () => {
+  let exportDest: string;
+
+  beforeAll(() => {
+    exportDest = tmpDir().name;
+  });
+
+  test("basic", async () => {
+    await runEngineTestV5(
+      async ({ engine, vaults, wsRoot }) => {
+        const pod = new GraphvizExportPod();
+        engine.config.useFMTitle = true;
```

Comment:
this should be true by default. is it not?

Reply:
To be perfectly honest, I kept this as it was the way a similar test was implemented. I can go ahead and remove it

Reply:
it wasn't default once upon a time ago. we should probably update the old tests :)

Reply:
Would you like me to take a look at the other engine tests and update them accordingly?